src/librustc_error_codes/error_codes/E0761.md

```diff
@@ -2,24 +2,20 @@ Multiple candidate files were found for an out-of-line module.
 
 Erroneous code example:
 
-```rust
+```ignore (multiple source files required for compile_fail)
```

Comment:
I never seen this, I thought `ignore` needs to be at the end?

Reply:
It's actually mandatory in error codes explanations. ;)